utils/tests.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/env bash
 
+set -e
 # `SECRET_PROJECT_ROOT` must be set before running the script.
+[ -z "${SECRET_PROJECT_ROOT}" ] && exit 1
 
-set -e
+function cleanup {
```

Comment:
I see these directories are created in tests/_test_base.bash. Can we rm them from there too? 

It's cleanest/clearest to have the same block of code be responsible for allocation and de-allocation (creation/destruction) and not another file of code.

Reply:
These are not the same directories. I made an error and used the same names. I can fix that. The TEST_TMP_DIR is for BATS to use as root tmp dir (each test make a subdir in that dir in _test_base_sh). The TEST_RUN_DIR is so that we can use full path when `rm -rf`

Reply:
ah, ok. Maybe some clarifiying comments are in order here too.